ledger/light/artifactmanager/handler.go

```diff
@@ -12,7 +12,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
+///
```

Comment:
The same

Reply:
already fixed. also i see in project somewhere using three dashes